Makefile

```diff
@@ -619,8 +620,19 @@ rpkg:
 	echo "import(Rcpp)" >> R-package/NAMESPACE
 	R CMD INSTALL R-package
 	Rscript -e "require(mxnet); mxnet:::mxnet.export('R-package'); warnings()"
-	rm R-package/NAMESPACE
+	$(RM) R-package/NAMESPACE
```

Comment:
Any reason for this change?

Reply:
for this Makefile a global remove command is defined, and it is reasonable to make consistent use of it